packages/calcite-components/src/components/input/input.tsx

```diff
@@ -1176,15 +1205,15 @@ export class Input
               accept={this.accept}
               aria-label={getLabelText(this)}
               autocomplete={this.autocomplete}
-              autofocus={this.el.autofocus ? true : null}
+              autofocus={autofocus}
               class={{
                 [CSS.editingEnabled]: this.editingEnabled,
                 [CSS.inlineChild]: !!this.inlineEditableEl,
               }}
               defaultValue={this.defaultValue}
               disabled={this.disabled ? true : null}
-              enterKeyHint={this.el.enterKeyHint}
-              inputMode={this.el.inputMode}
+              enterKeyHint={enterKeyHint}
+              inputMode={inputMode}
```

Comment:
Yeah this makes more sense. We'll just have to put the previous code back once we remove the internal props.